barretenberg/cpp/src/barretenberg/stdlib/primitives/bigfield/bigfield.fuzzer.hpp

```diff
@@ -266,7 +266,7 @@ template <typename Builder> class BigFieldBase {
                 mask = (uint256_t(1) << mask_size) - 1;
                 // Choose the bit range
                 // Return instruction
-                return { .id = instruction_opcode, .arguments.element = Element(temp & mask) };
+                return { .id = instruction_opcode, .arguments.element = Element(static_cast<uint64_t>(temp & mask)) };
```

Comment:
I also see this same line in field.fuzzer.hpp; no need to change there too?

Reply:
I guess it wasn't used, and we'll hit errors when we try to use it.